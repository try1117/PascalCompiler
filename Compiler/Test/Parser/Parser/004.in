program abc;
type
 myInteger = integer;
 myDouble = double; 
begin
end.