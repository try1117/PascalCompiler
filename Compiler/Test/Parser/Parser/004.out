abc : function()
   resultType : Nil
   
Declarations:
   myInteger : Type Integer
   myDouble : Type Double
